packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/reconnect-edge/useReconnectEdge.tsx

```diff
@@ -71,9 +71,11 @@ export const useReconnectEdge = (): UseReconnectEdge => {
         reconnectEdgeKind,
         newEdgeEndPosition: { x: 0, y: 0 },
       };
-      updateEdgeEnd({ variables: { input } });
+      if (!readOnly) {
```

Comment:
Given that edges should not be updatable in read only, I suspect that this is not possible either